include/code_080023A4.h

```diff
@@ -0,0 +1,19 @@
+#ifndef GUARD_CODE_080023A4_H
+#define GUARD_CODE_080023A4_H
+
+#include "global.h"
+
+struct Unk_02023530 {
+	u8 unk000[484];
+	void **objlist_ptr; // 0x1E4
+	u8 unk1E8[1040];
+	u16 current_room; // 0x5F8
+	u8 unk5FA[110];
+};
+
+extern struct Unk_02023530 gUnk_02023530[4];
+extern u32 gUnk_0203AD10;
```

Comment:
move them to include/data.h. Note that there's a large gap between the symbols, which implies they're almost impossible to be initialized in the same source file.